### PR TITLE
feat : added grid and slider components

### PR DESCRIPTION
**Pull request by ArjunJunna**
Added the grid and slider components.

### CODE REVIEW

**Review by sanathkumar26700 — APPROVED** [review]
Everything working fine, code is fine.

File: components/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.grid-layout{
+  display: grid;
+  grid-template-rows: 1fr;
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
```
Comment (mikesch803) [inline components/grid/grid.css]: suggestion - if u are not giving gaps, remove these properties

**Review by mikesch803 — CHANGES_REQUESTED** [review]
code is clean, great work! I put a suggestion

File: components/grid/grid.css (hunk 1 above)
Comment (ArjunJunna) [reply components/grid/grid.css]: done made the changes thank you

**Pull request merged**